SOOP/SOOP_CO2/incoming_handler.sh

```diff
@@ -79,6 +81,9 @@ main() {
         s3_put_no_index $extracted_file $path/$file_basename
     done
 
+    echo "" | notify_by_email $recipient "Successfully published SOOP_CO2 file '$path' "
+    echo "" | notify_by_email $BACKUP_RECIPIENT "Successfully published SOOP_CO2 file '$path'"
+
```

Comment:
I agree, it is ok cause i know i only get a few files. And again yes, having a single report at the end of the process is the way to go.